tests/support/commands.ts

```diff
@@ -43,12 +43,27 @@ Cypress.Commands.add('installRuntimeConfig', ({ type } = {}): void => {
   --cos_password=minioadmin \
   --cos_bucket=test-bucket';
 
+ 
   cy.exec(
     type === 'kfp' ? kfpRuntimeInstallCommand : airflowRuntimeInstallCommand,
     { failOnNonZeroExit: false }
   );
 });
 
+
+const brokenAirFlowRunTimeInstallCommand =
```

Comment:
This command can be removed as discussed, not being used. This should also get rid of the lint failure message

> /home/runner/work/elyra/elyra/tests/support/commands.ts
>   54:7  warning  'brokenAirFlowRunTimeInstallCommand' is assigned a value but never used  @typescript-eslint/no-unused-vars